template-manager-cmtemplate/src/main/java/com/sequenceiq/cloudbreak/cmtemplate/configproviders/atlas/AtlasKnoxRoleConfigProvider.java

```diff
@@ -33,6 +33,8 @@ public class AtlasKnoxRoleConfigProvider extends AbstractRoleConfigProvider {
 
     public static final String MIN_VERSION_FOR_DIFFERENCIAL_BACKUP = "7.2.7";
 
+    public static final String MAX_VERSION_FOR_DIFFERENCIAL_BACKUP = "7.2.17";
```

Comment:
This and above should say `DIFFERENTIAL`.

Reply:
MAX_VERSION_FOR_DIFFERENCIAL_BACKUP would mean the version till which differential backup is supported.
In this case it should be 7.2.16.